From dfbff6274ce25814a4357a8a788c404425cecb69 Mon Sep 17 00:00:00 2001
From: orbea <orbea@fredslev.dk>
Date: Sat, 1 Dec 2018 07:43:12 -0800
Subject: [PATCH] Makefile: Add MANDIR.

Not all Linux distros install manpages to $(PREFIX)/share/man.
For example Slackware installs manpages to /usr/man/.
---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 6010bc9..4cb7093 100644
--- a/Makefile
+++ b/Makefile
@@ -13,6 +13,7 @@
 PREFIX ?= /usr
 INSTALL ?= install
 LIBDIR ?= lib
+MANDIR ?= share/man
 
 nls ?= 1
 xcb ?= 1
@@ -103,7 +104,7 @@ install: all
 ifeq ($(xcb), 1)
 	$(INSTALL) -D -m755 $(xcblib) $(DESTDIR)/$(PREFIX)/$(LIBDIR)/$(xcblib)
 endif
-	$(INSTALL) -D -m644 radeontop.1 $(DESTDIR)/$(PREFIX)/share/man/man1/radeontop.1
+	$(INSTALL) -D -m644 radeontop.1 $(DESTDIR)/$(PREFIX)/$(MANDIR)/man1/radeontop.1
 ifeq ($(nls), 1)
 	$(MAKE) -C translations install PREFIX=$(PREFIX)
 endif
